frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -435,22 +435,27 @@ const ReceiverWrapper: React.FC<ReceiverFormsWrapperProps> = React.memo(({ obj,
       setLoadError({ message: `Error alertManagerBaseURL not set` });
       return;
     }
-    coFetchJSON(`${alertManagerBaseURL}/api/v2/status/`).then((data) => {
-      const originalAlertmanagerConfigJSON = data?.config?.original;
-      if (_.isEmpty(originalAlertmanagerConfigJSON)) {
-        setLoadError({ message: 'alertmanager.v2.status.config.original not found.' });
-      } else {
-        try {
-          const { global } = safeLoad(originalAlertmanagerConfigJSON);
-          setAlertmanagerGlobals(global);
-          setLoaded(true);
-        } catch ({ message }) {
-          setLoadError({
-            message: `Error parsing Alertmanager config.original: ${message || 'invalid YAML'}`,
-          });
+    coFetchJSON(`${alertManagerBaseURL}/api/v2/status/`)
+      .then((data) => {
+        const originalAlertmanagerConfigJSON = data?.config?.original;
+        if (_.isEmpty(originalAlertmanagerConfigJSON)) {
+          setLoadError({ message: 'alertmanager.v2.status.config.original not found.' });
+        } else {
+          try {
+            const { global } = safeLoad(originalAlertmanagerConfigJSON);
+            setAlertmanagerGlobals(global);
+            setLoaded(true);
+          } catch ({ message }) {
+            setLoadError({
+              message: `Error parsing Alertmanager config.original: ${message || 'invalid YAML'}`,
+            });
+          }
         }
-      }
-    });
+      })
+      .catch((e) => {
+        e.message = `Error loading ${alertManagerBaseURL}/api/v2/status/: ${e}`;
+        setLoadError(e);
```

Comment:
No reason to mutate `e`

```suggestion
        setLoadError({ message: `Error loading ${alertManagerBaseURL}/api/v2/status/: ${e.message}` });
```

Reply:
Fixed.  I had it that way originally, but we lose the 'Error:' in '...Error: Forbidden', but I think it's ok:
![image](https://user-images.githubusercontent.com/12733153/73392658-3b985780-42a8-11ea-830b-4b57afd72919.png)

Reply:
Fixed